### PR TITLE
Add scipy 2024 news item and update "upcoming"

### PR DESCRIPTION
**Pull request by jat255**

### CODE REVIEW

File: conf.py [hunk 1]
```diff
@@ -30,6 +30,13 @@
 
 linkcheck_ignore = [
     "https://sbmm.org.br/en/28o-csbmm",  # 403 Client Error: Forbidden for url
+    "https://www.diamond.ac.uk/Home/Events/2020/HyperSpy_2020.html",  # - 403 Client Error: Forbidden for url: https://www.diamond.ac.uk/Home/Events/2020/HyperSpy_2020.html
+    "https://www.diamond.ac.uk/Home/Events/2018/HyperSpy.html",  # - 403 Client Error: Forbidden for url: https://www.diamond.ac.uk/Home/Events/2018/HyperSpy.html
+    "https://www.diamond.ac.uk/Home/Events/2021/HyperSpy_2021.html",  # - 403 Client Error: Forbidden for url: https://www.diamond.ac.uk/Home/Events/2021/HyperSpy_2021.html
+    "https://www.diamond.ac.uk/Home/Events/2022/HyperSpy_2022.html",  # - 403 Client Error: Forbidden for url: https://www.diamond.ac.uk/Home/Events/2022/HyperSpy_2022.html
+    "https://www.diamond.ac.uk/Home/Events/2019/HyperSpy_2019.html",  # - 403 Client Error: Forbidden for url: https://www.diamond.ac.uk/Home/Events/2019/HyperSpy_2019.html
+    "https://www.diamond.ac.uk/Home/Events/2024/HyperSpy-Workshop",  #-2024.html - 403 Client Error: Forbidden for url: https://www.diamond.ac.uk/Home/Events/2024/HyperSpy-Workshop-2024.html
```
Comment (jlaehne) [inline conf.py:38]:
```suggestion
    "https://www.diamond.ac.uk/Home/Events/2024/HyperSpy-Workshop-2024.html",  # - 403 Client Error: Forbidden for url: https://www.diamond.ac.uk/Home/Events/2024/HyperSpy-Workshop-2024.html
```

File: index.rst [hunk 2]
```diff
@@ -297,7 +297,7 @@
         <https://zenodo.org>`_ that can be found in the respective :ref:`documentations 
         <ecosystem_label>`. DOIs for specific 
         versions of HyperSpy can be found by clicking on the `Concept DOI 
-        <https://help.zenodo.org/faq/#versioning>`_ button.
+        <https://support.zenodo.org/help/en-gb/1-upload-deposit/97-what-is-doi-versioning>`_ button.
```
Comment (jlaehne) [inline index.rst:300]: Hm, why do they always change that link, we might have to change it in a number of documentations as well.

**Review by jlaehne — APPROVED** [review]

**Comment by jlaehne** [comment]
@ericpre Any idea why the epsic links were failing? They did always run through previously. Not sure if we should indeed excempt them from the link check.

**Comment by jlaehne** [comment]
@jat255 was there ever a publication of the videos from IUMAS? I remember there was a sound issue for the initial ones.

File: conf.py [hunk 3]
```diff
@@ -30,6 +30,13 @@
...
+    "https://www.diamond.ac.uk/Home/Events/2018/HyperSpy.html",  # - 403 Client Error: Forbidden for url: https://www.diamond.ac.uk/Home/Events/2018/HyperSpy.html
+    "https://www.diamond.ac.uk/Home/Events/2021/HyperSpy_2021.html",  # - 403 Client Error: Forbidden for url: https://www.diamond.ac.uk/Home/Events/2021/HyperSpy_2021.html
+    "https://www.diamond.ac.uk/Home/Events/2022/HyperSpy_2022.html",  # - 403 Client Error: Forbidden for url: https://www.diamond.ac.uk/Home/Events/2022/HyperSpy_2022.html
+    "https://www.diamond.ac.uk/Home/Events/2019/HyperSpy_2019.html",  # - 403 Client Error: Forbidden for url: https://www.diamond.ac.uk/Home/Events/2019/HyperSpy_2019.html
+    "https://www.diamond.ac.uk/Home/Events/2024/HyperSpy-Workshop-2024.html",  # - 403 Client Error: Forbidden for url: https://www.diamond.ac.uk/Home/Events/2024/HyperSpy-Workshop-2024.html
+    "https://www.diamond.ac.uk/Home/Events/2023/HyperSpy_2023.html",  # - 403 Client Error: Forbidden for url: https://www.diamond.ac.uk/Home/Events/2023/HyperSpy_2023.html
```
Comment (ericpre) [inline conf.py:39]:
```suggestion
    "https://www.diamond.ac.uk/Home/Events",  # - 403 Client Error: Forbidden for url: https://www.diamond.ac.uk/Home/Events/2023/HyperSpy_2023.html
```

**Comment by jat255** [comment]
> @jat255 was there ever a publication of the videos from IUMAS? I remember there was a sound issue for the initial ones.

No, I don't think we ever got those up anywhere, but I still have the videos and will get to it soon enough!

**Pull request merged**